### PR TITLE
support bundle loading per FW

### PR DESCRIPTION
**Pull request by oren-zohar**
Adding bundle configuration support per framework (Vanilla and EKS)

### CODE REVIEW

File: evaluator/bundle.go [hunk 1]
```diff
@@ -35,15 +35,19 @@ var (
 )
 
 func StartServer(ctx context.Context, cfg config.Config) (*http.Server, error) {
-	bundle := csppolicies.CISKubernetesBundle()
+	log := logp.NewLogger("cloudbeat_bundle_server")
+
```
Comment (ofiriro3) [inline evaluator/bundle.go:39]: nit - no space is needed here

File: evaluator/bundle.go [hunk 2]
```diff
@@ -69,3 +71,18 @@ func StartServer(ctx context.Context, cfg config.Config) (*http.Server, error) {
 
 	return srv, nil
 }
+
+// loadBundle loads the relevant bundle based on the config.
+func loadBundle(cfg config.Config, log *logp.Logger) csppolicies.Bundle {
```
Comment (ofiriro3) [inline evaluator/bundle.go:76]: Wondering about this method.

Do we wish to throw an error if we don't receive any of the two?

WDT?

File: evaluator/bundle_test.go [hunk 3]
```diff
@@ -94,17 +94,10 @@ func (s *BundleTestSuite) TestCreateServerWithDataYaml() {
...
     streams:
-      - data_yaml:
-          activated_rules:
-            cis_k8s:
-              - a
-              - b
-              - c
-              - d
-              - e
+      - data_yaml
```
Comment (ofiriro3) [inline evaluator/bundle_test.go:97]: Was the test changed on purpose?

File: evaluator/bundle_test.go [hunk 4]
```diff
@@ -170,3 +163,77 @@ func (s *BundleTestSuite) TestCreateServerWithDataYaml() {
...
+
+	time.Sleep(time.Second * 2)
+	for _, test := range tests {
+		s.Run(test.name, func() {
+			server, err := StartServer(context.Background(), test.cfg)
+			s.NoError(err)
+
+			target := ServerAddress + test.path
+			client := &http.Client{}
+			res, err := client.Get(target)
```
Comment (ofiriro3) [inline evaluator/bundle_test.go:230]: Not a clastic unit test, as far as I understand it uses a real HTTP client. 
I would have mocked it, but you relied on others' work.

Definitely not a must.

**Review by ofiriro3 — CHANGES_REQUESTED** [review]
Good job. 🦑 

Few small comments + I would love to hear your thoughts on one of the questions I left

File: evaluator/bundle_test.go (hunk 3 above)
Comment (oren-zohar) [reply evaluator/bundle_test.go:97]: yeah the test was wrong

File: evaluator/bundle.go (hunk 2 above)
Comment (oren-zohar) [reply evaluator/bundle.go:76]: good point. 
I had the impression that we would want to have vanilla as default in any case the config has some misconfiguration as we had before.
Additionally, keeping it like that will be backward compatible with the previous config format, which is still what we have today, and will not break the integration test. We can update this function to be more strict once we have working integration between the config and cloudbeat.

**Review by ofiriro3 — APPROVED** [review]
🚀

**Pull request merged**